pkg/api/defaults.go

```diff
@@ -308,7 +308,7 @@ func (p *Properties) setExtensionDefaults() {
 func (p *Properties) setMasterProfileDefaults(isUpgrade bool) {
```

Comment:
Do we need the conversion of deprecated distros used by master vms during upgrade in this func?

Reply:
whoops yes you're right... I guess `AKSDockerEngine` didn't need it because it was an agent only distro

Reply:
done